### PR TITLE
Epmrpp 33934

### PR DESCRIPTION
**Pull request by AntonBotnykov**

### CODE REVIEW

File: app/localization/translated/be.json [hunk 1]
```diff
@@ -47,6 +47,12 @@
   "FiltersPage.msgFavoriteFilters": "Фільтры - пошукавыя запыты, якія былі ўпадабаны для паўторнага выкарыстання.",
   "FiltersPage.searchByName": "Пошук па назве",
   "FiltersPage.title": "Фільтры",
+  "Footer.build": "Build",
```
Comment (ArtsiomYeliseyenka) [inline app/localization/translated/be.json]: "Зборка"

File: app/src/components/main/scrollWrapper/scrollWrapper.scss [hunk 2]
```diff
@@ -37,4 +37,6 @@
   transition: opacity 400ms
 }
 .scrolling-content {
+  display: flex;
+  flex-direction: column;
```
Comment (ArtsiomYeliseyenka) [inline app/src/components/main/scrollWrapper/scrollWrapper.scss]: These changes breaks another components which are using scrollWrapper component. Do we really need to change scrollWrapper styles?

File: app/src/controllers/appInfo/selectors.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 export const appInfoSelector = state => state.appInfo || {};
 
+export const appInfoBuildVersion = state => appInfoSelector(state).UI.build.version || {};
+
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/appInfo/selectors.js]: Do not create additional selector. To get UI version use existing appInfoSelector directly in footer component.

Comment (DiscoElevator) [reply app/src/controllers/appInfo/selectors.js]: Why not? Dedicated selector makes it easier to access data.
I just suggest to rename it to `buildVersionSelector`.

**Comment by AntonBotnykov** [comment]
done all requested changes

**Comment by AntonBotnykov** [comment]
done all requested changes

File: app/src/layouts/appLayout/footer/footer.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+import { buildVersionSelector } from 'controllers/appInfo';
+import { referenceDictionary } from 'common/utils';
+import styles from './footer.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(state => ({
+  buildVersion: buildVersionSelector(state),
+}))
+export class Footer extends PureComponent {
```
Comment (DiscoElevator) [inline app/src/layouts/appLayout/footer/footer.jsx]: Connected components are pure by default, so no need to use `PureComponent` here

Comment (AntonBotnykov) [reply app/src/layouts/appLayout/footer/footer.jsx]: done

File: app/localization/translated/be.json [hunk 5]
```diff
@@ -63,6 +63,20 @@
   "FiltersPage.msgFavoriteFilters": "Фільтры - пошукавыя запыты, якія былі ўпадабаны для паўторнага выкарыстання.",
   "FiltersPage.searchByName": "Пошук па назве",
   "FiltersPage.title": "Фільтры",
+  "Footer.build": "Зборка",
+  "Footer.copyright": "Усе правы абаронены.",
+  "Footer.git": "Адгалінаваць праект",
+  "Footer.slack": "Паразмаўляйце з намі у Slack",
+  "Footer.contact": "Звяжыцеся з намі",
+  "Footer.documentation": "Дакументацыя",
+  "DurationBlock.finished": "Працягласць: {durationTime}. Час заканчэння: {endTime}",
```
Comment (DiscoElevator) [inline app/localization/translated/be.json]: Duplicate entries for `DurationBlock`

**Pull request merged**